threedi_api_qgis_client/metadata.txt

```diff
@@ -0,0 +1,47 @@
+# This file contains metadata for your plugin.
+
+# This file should be included when you package your plugin.# Mandatory items:
+
+[general]
+name=3Di API Client
+qgisMinimumVersion=3.4
+description=3Di API Client for QGIS
+version=0.1
+author=Lutra Consulting for 3Di Water Management
+email=info@lutraconsulting.co.uk
+
+about=3Di API Client for QGIS
+
+tracker=https://github.com/nens/threedi-api-qgis-client/issues
+repository=https://github.com/nens/threedi-api-qgis-client
+# End of mandatory metadata
+
+# Recommended items:
+
+hasProcessingProvider=no
+# Uncomment the following line and add your changelog:
+# changelog=
+
+# Tags are comma separated with spaces allowed
+tags=python
```

Comment:
would be good to add 3Di, Water Modelling, etc...

Reply:
Done.